README.md

```diff
@@ -22,28 +22,60 @@ Our project consists of two main parts. The first part of our project is data vi
 * Maeve Murphy
 * Reagan Smith
 
-## MVP objective
+## Architecture
+
+All data processing is completed using Python 3. Ruby will use the Python scripts to display the resulting graphs on web pages, titled "Overall Distribution" and "Individual Statistics."
+
+### Database Design
+
+The database is implemented using SQLite, and follows this structure:
+
+### Prerequisites
 
-For our minimum viable product (MVP), we expect to deliver two main functionalities: a method to import survey data and a basic graphical representation of the data. To import survey data, we plan on allowing the user to enter a json with multiple people’s results from the survey. This data will then be stored in our database. The second part of our MVP is graphical interpretations of the data. For the MVP, we plan on visualizing the overall results of the data so that the user can see the proportions of challenger to communicator to collaborator to contributor for the class. We also plan to have a set of graphs that show a person’s individual scores and a set of graphs that will show the distribution of responses to each question. 
+This application requires:
+* Python 3.x
+* Rails 5 with Ruby 2.2 or higher
+    - Can be installed by following [these](http://railsapps.github.io/installing-rails.html) instructions
+* Git
+
+### Gems
+
+* High Voltage [found here](https://github.com/thoughtbot/high_voltage)
+
+## Installation
+
+To use this app, you must first download the files. To do so, open up a terminal on your personal machine, navigate to the folder in which you'd like the files to exist, and run these commands
+- `git clone https://github.com/nathanljustin/teamwork-analysis.git`
+- `cd teamworkApp`
+- `rails db:migrate`
+- `rails s`
+
+Then go to your web browser of choice, and navigate to `localhost:3000`. You are now free to use the teamwork analysis app!
 
 ## Functionality
 
-* We will provide an interface where, after submitting their survey, users can see any relevant charts, graphs, and information.
-* We will have an analysis of survey results that is useful in helping students find compatible teammates and interesting patterns or characteristics in the data that the instructor can use. The analysis will be backed up by research.
-* We will provide algorithms that can interpret the data in multiple ways.
+* Import Data: The foundation of our Teamwork Survey web app is the data, so we must have an easy way to put data in the database. When a user starts using the app, the database will be empty. The user’s only option will be to import data from a .csv file into the database, via a button on the main page. The web app assumes that the .csv file is set up such that it was an output from the [Teamwork Survey](https://docs.google.com/forms/d/1px6yEtC9zEG469OELr_1apL2iTUqwwDRYkoirncgVBI/edit) sent out by the muddersOnRails team. 
```

Comment:
Suggestion:  change "When a user starts using the app, the database will be empty." to "Upon first loading the web app, the database will be empty."